### PR TITLE
[stable/prometheus] Added remote_write and remote_read for prometheus

### PR DESCRIPTION
**Pull request by sergeimonakhov**
#### What this PR does / why we need it:
Adds the ability to specify remote_write and remote_read in the prometheus configuration

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@D1abloRUS 
Would you mind to fix two DCO signoff please?
Commit sha: c54e514, Author: Sergey Monakhov, Committer: Sergey Monakhov; The sign-off is missing.
Commit sha: d10199f, Author: Sergey Monakhov, Committer: Sergey Monakhov; The sign-off is missing.

**Review by zanhsieh — APPROVED** [review]

File: stable/prometheus/templates/server-configmap.yaml [hunk 1]
```diff
@@ -13,6 +13,14 @@ data:
 {{- if eq $key "prometheus.yml" }}
     global:
 {{ $root.Values.server.global | toYaml | trimSuffix "\n" | indent 6 }}
+{{- if $root.Values.server.remoteWrite }}
+    remote_write:
+{{ $root.Values.server.remoteWrite | toYaml | indent 4 }}
```
Comment (flecno) [inline stable/prometheus/templates/server-configmap.yaml:18]: Isn't indent of 6 required here? Also for remote_read

**Pull request merged**